### PR TITLE
MM-33629 - HPNS docs are outdated

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-33629

Updated:
- Self-Managed Admin Guide > Mobile Apps > Hosted Push Notification Service
   - Updated procedural steps under "Setting up HPNS push notifications in Enterprise Edition"
   - Applied formatting and clarification updated

### CODE REVIEW

File: source/mobile/mobile-hpns.rst [hunk 1]
```diff
@@ -8,37 +8,37 @@ When using Mattermost mobile apps in the Apple App Store and Google Play, purcha
...
 Our Hosted Push Notification Service offers:
 
-  - Access to a publicly-hosted Mattermost Push Notification Service (MPNS) offering an explicit `privacy policy <https://mattermost.com/data-processing-addendum/>`__ where the contents of unencrypted messages are not examined or stored
-  - Encrypted TLS connections between HPNS and Apple Push Notification Services, HPNS and Google’s Firebase Cloud Messaging service, and HPNS and your Mattermost server
-  - Production-level uptime expectation
+- Access to a publicly-hosted Mattermost Push Notification Service (MPNS) offering an explicit `privacy policy <https://mattermost.com/data-processing-addendum/>`__ where the contents of unencrypted messages are not examined or stored.
+- Encrypted TLS connections between HPNS and Apple Push Notification Services, HPNS and Google’s Firebase Cloud Messaging service, and HPNS and your Mattermost server.
+- Production-level uptime expectation.
 
 After purchasing a subscription to Mattermost E10 or higher from Mattermost, Inc. follow the instructions below to set up and test your system.
```
Comment (justinegeffen) [inline source/mobile/mobile-hpns.rst:15]:
```suggestion
After purchasing a subscription to Mattermost Enterprise Edition E10 or higher from Mattermost, Inc. follow the instructions below to set up and test your system.
```

File: source/mobile/mobile-hpns.rst [hunk 2]
```diff
@@ -8,37 +8,37 @@ When using Mattermost mobile apps in the Apple App Store and Google Play, purcha
...
 
-.. Note:: Both TPNS and HPNS only work with the Mattermost Apple App Store and Google Play apps. If you have compiled the apps yourselves, you must also host your own Mattermost push proxy server. See our FAQ on :ref:`how push notifications work <push-faq>` for more details. Our push proxy server is `available on Github. <https://github.com/mattermost/mattermost-push-proxy>`__
+.. Note:: 
 
-If you use HPNS you need to ensure that the push proxy can be reached on port 443 from the Mattermost server, for TPNS it is port 80. If you host your own proxy server the default port is 8086.
+  - Both TPNS and HPNS only work with the Mattermost Apple App Store and Google Play apps. If you have compiled the apps yourself, you must also host your own Mattermost push proxy server. See our FAQ on :ref:`how push notifications work <push-faq>` for more details. Our push proxy server is `available on Github. <https://github.com/mattermost/mattermost-push-proxy>`__
```
Comment (justinegeffen) [inline source/mobile/mobile-hpns.rst:19]:
```suggestion
  - Both TPNS and HPNS only work with the Mattermost Apple App Store and Google Play apps. If you have compiled the apps yourself, you must also host your own Mattermost push proxy server. See our FAQ on :ref:`how push notifications work <push-faq>` for more details. Our push proxy server is `available on GitHub. <https://github.com/mattermost/mattermost-push-proxy>`__
```

File: source/mobile/mobile-hpns.rst [hunk 3]
```diff
@@ -8,37 +8,37 @@ When using Mattermost mobile apps in the Apple App Store and Google Play, purcha
...
+Setting up HPNS Push Notifications in Enterprise Edition
 --------------------------------------------------------
 
 Follow these steps to set up HPNS:
 
-1. Follow the instructions to `install or upgrade to Enterprise Edition <https://docs.mattermost.com/install/ee-install.html>`__
+1. Follow the instructions to `install or upgrade to Enterprise Edition <https://docs.mattermost.com/install/ee-install.html>`__.
 
-2. Go to **System Console** > **Notifications** > **Mobile Push** > **Send Push Notifications** in prior versions or **System Console > Environment > Push Notification Server** in versions after 5.12
+2. Go to **System Console > Environment > Push Notification Server** (or go to **System Console > Notifications > Mobile Push > Send Push Notifications** in versions prior to v5.12).
```
Comment (justinegeffen) [inline source/mobile/mobile-hpns.rst:30]:
```suggestion
2. Go to **System Console > Environment > Push Notification Server**.
```

File: source/mobile/mobile-hpns.rst [hunk 4]
```diff
@@ -8,37 +8,37 @@ When using Mattermost mobile apps in the Apple App Store and Google Play, purcha
...
 
-3. Select "Use encrypted, production-quality HPNS connection to iOS and Android apps" (this option appears only in Enterprise Edition)
+3. Set **Enable Push Notifications** to **Use HPNS connection with uptime SLA to send notifications to iOS and Android apps**. Note that this option appears only in Enterprise Edition.
```
Comment (justinegeffen) [inline source/mobile/mobile-hpns.rst:32]:
```suggestion
3. Set **Enable Push Notifications** to **Use HPNS connection with uptime SLA to send notifications to iOS and Android apps**. Note that this option is only available in Enterprise Edition.
```

**Review by justinegeffen — APPROVED** [review]
Some formatting and wording suggestions. Non-blocking but would be beneficial.

**Review by adamjclarkson — APPROVED** [review]

**Pull request merged**